test/helpers/ssh_command.go

```diff
@@ -248,30 +248,62 @@ func (client *SSHClient) RunCommandContext(ctx context.Context, cmd *SSHCommand)
 		panic("nil context provided to RunCommandContext()")
 	}
 
-	session, err := client.newSession()
-	if err != nil {
-		return err
-	}
+	var (
+		session    *ssh.Session
+		err        error
+		sessionErr error
+	)
+
+	sessionErrChan := make(chan error, 1)
```

Comment:
Can be between lines 254 and 255 (as part of the `var`)

Reply:
Done.